apps/services/auth/delegation-api/src/app/delegations/meDelegations.controller.ts

```diff
@@ -180,8 +180,8 @@ export class MeDelegationsController {
   findOne(
     @CurrentUser() user: User,
     @Param('delegationId') delegationId: string,
-  ): Promise<DelegationDTO> {
-    return this.delegationsOutgoingService.findById(user, delegationId)
+  ): Promise<DelegationDTO | null> {
+    return this.delegationsService.findById(user, delegationId)
```

Comment:
I can't do this due to the different scope access handling. So I'm going to extract those auth-api changes to separate PR over the weekend and make a fix.